s3/tests/unit.tftest.hcl

```diff
@@ -1,7 +1,7 @@
 variables {
   vpc_name    = "s3-test-vpc-name"
   application = "s3-test-application"
-  environment = "s3-test-environment"
+  environment = "s3-test-environmnent"
```

Comment:
Surely we fix the source where the misspelling occurs, not all the instances where it's checked?

Reply:
Actually, why have you introduced the misspelling?